### ISSUE TITLE
Fails completely if file is blocked as malware

### ISSUE THREAD

**Issue by Fabian42** [cjnaz/RCloneSync#19]
Google Drive is stupid, so it blocks backups of apps like Titanium Backup and XPosed installer as malware. (report here: https://issuetracker.google.com/issues/77781871)
For some reason that means that I can still upload them, but not download them.
Currently this makes RCloneSync fail completely, because it can't download the file.

**Comment by cjnaz** [comment]
Interesting.  I use Titanium Backup also, but back up to Dropbox and have RCloneSync sync the backup folder to my local drive.

There may be hope.  I uploaded your com.keramidas.TitaniumBackupPro-721447d4e65419eb3b3c37fa79ca15e2.apk.bz2 file to my Google Drive folder via the web interface and then ran RCloneSync.  It worked as expected.  No errors.  Then, via the Google Drive web interface I tried to download the file and got the "... is infected with a virus" message, with a DOWNLOAD INFECTED FILE. anyway option which worked.  Obviously rclone can't mash an "I know, download it anyway" button.  

Next, I renamed the file via the web interface and on my local machine ran rclone copy, which worked as expected
`[<me>@<machine> TiBu]$ rclone copy GDrive:/TiBu/com.Xkeramidas.TitaniumBackupPro-721447d4e65419eb3b3c37fa79ca15e2.apk.bz2 .`

Question RE "Fails completely" - What does RCloneSync do when this happens?  Does it crash, or exit Critical (requiring a --FirstSync), or exit with Error (allowing rerun)?  I expect that it exits with Critical error since it can't make the local filesystem the master.

I suggest posting the problem to the rclone forum.  It may be an issue with your version of rclone.  Again, I'm on `rclone v1.37-157-g5a3a56ab<beta>`.

**Comment by Fabian42** [comment]
Yes, it exits with critical error. I'll test with rclone alone, once I find
time (so much debugging and setting up to do this weekend).

**Comment by Fabian42** [comment]
It did successfully run with rclone alone. But since some of my problems
have started to become random, I'll do some more tests.

**Comment by Fabian42** [comment]
Nope, it also gave an error message now. Despite actually downloading it.

**Comment by Fabian42** [comment]
RClone beta has a `--drive-acknowledge-abuse` flag now, this would have to be somehow integrated into RCloneSync.
